diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x ltmain.sh -x 'config.*' -x depcomp -x install-sh -x missing -x mkinstalldirs -x autom4te.cache -x '*compile' -x 'intltool*.in' -x 'xml-i18n-*.in' -x '*.pyc' -x '*.mo' -x '*.gmo' -x ABOUT-NLS -x Makevars.template -x COPYING -x INSTALL -x '*.orig' -x '*.rej' -x '*~' -x '*.temp' -x texinfo.tex -x ylwrap -x 'Makefile.in*' -x 'makefile.in*' -x configure -x gnome-doc-utils.make -x intltool.m4 -x omf.make -x xmldocs.make -x common -x m4 -x po -x autoregen.sh -x stamp-h.in origsrc/gst-editor-0.8.0/autogen.sh src/gst-editor-0.8.0/autogen.sh
--- origsrc/gst-editor-0.8.0/autogen.sh	1969-12-31 18:00:00.000000000 -0600
+++ src/gst-editor-0.8.0/autogen.sh	2006-07-21 17:40:32.863625000 -0500
@@ -0,0 +1,92 @@
+#!/bin/sh
+# Run this to generate all the initial makefiles, etc.
+
+DIE=0
+package=gst-editor
+srcfile=src/editor.c
+
+# a quick cvs co if necessary to alleviate the pain - may remove this
+# when developers get a clue ;)
+if test ! -d common; 
+then 
+  echo "+ getting common/ from cvs"
+  cvs co common 
+fi
+
+# source helper functions
+if test ! -f common/gst-autogen.sh;
+then
+  echo There is something wrong with your source tree.
+  echo You are missing common/gst-autogen.sh
+  exit 1
+fi
+. common/gst-autogen.sh
+
+CONFIGURE_DEF_OPT='--enable-maintainer-mode --enable-debug --enable-DEBUG'
+
+autogen_options $@
+
+echo -n "+ check for build tools"
+if test ! -z "$NOCHECK"; then echo " skipped"; else  echo; fi
+version_check "autoconf" "$AUTOCONF autoconf autoconf-2.54 autoconf-2.53 autoconf-2.52" \
+              "ftp://ftp.gnu.org/pub/gnu/autoconf/" 2 52 || DIE=1
+version_check "automake" "$AUTOMAKE automake automake-1.7 automake-1.6 automake-1.5" \
+              "ftp://ftp.gnu.org/pub/gnu/automake/" 1 5 || DIE=1
+version_check "gettext" "" \
+              "ftp://alpha.gnu.org/gnu/" 0 10 35 || DIE=1
+version_check "intltoolize" "" \
+              "ftp://ftp.gnome.org/pub/GNOME/stable/sources/intltool" 0 1 5 || DIE=1
+version_check "libtool" "" \
+              "ftp://ftp.gnu.org/pub/gnu/libtool/" 1 4 0 || DIE=1
+version_check "pkg-config" "" \
+              "http://www.freedesktop.org/software/pkgconfig" 0 8 0 || DIE=1
+
+die_check $DIE
+
+autoconf_2_52d_check || DIE=1
+aclocal_check || DIE=1
+autoheader_check || DIE=1
+
+die_check $DIE
+
+# if no arguments specified then this will be printed
+if test -z "$*"; then
+  echo "+ checking for autogen.sh options"
+  echo "  This autogen script will automatically run ./configure as:"
+  echo "  ./configure $CONFIGURE_DEF_OPT"
+  echo "  To pass any additional options, please specify them on the $0"
+  echo "  command line."
+fi
+
+toplevel_check $srcfile
+
+# FIXME : why does libtoolize keep complaining about aclocal ?
+tool_run "glib-gettextize" "--copy --force"
+tool_run "intltoolize" "--copy --force --automake"
+tool_run "libtoolize" "--copy --force"
+tool_run "$aclocal" "-I m4 -I common/m4 $ACLOCAL_FLAGS"
+tool_run "$autoheader"
+
+# touch the stamp-h.in build stamp so we don't re-run autoheader in maintainer mode -- wingo
+echo timestamp > stamp-h.in 2> /dev/null
+
+tool_run "$autoconf"
+tool_run "$automake" "-a -c"
+
+test -n "$NOCONFIGURE" && {
+  echo "+ skipping configure stage for package $package, as requested."
+  echo "+ autogen.sh done."
+  exit 0
+}
+
+echo "+ running configure ... "
+test ! -z "$CONFIGURE_DEF_OPT" && echo "  ./configure default flags: $CONFIGURE_DEF_OPT"
+test ! -z "$CONFIGURE_EXT_OPT" && echo "  ./configure external flags: $CONFIGURE_EXT_OPT"
+echo
+
+./configure $CONFIGURE_DEF_OPT $CONFIGURE_EXT_OPT || {
+        echo "  configure failed"
+        exit 1
+}
+
+echo "Now type 'make' to compile $package."
diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x ltmain.sh -x 'config.*' -x depcomp -x install-sh -x missing -x mkinstalldirs -x autom4te.cache -x '*compile' -x 'intltool*.in' -x 'xml-i18n-*.in' -x '*.pyc' -x '*.mo' -x '*.gmo' -x ABOUT-NLS -x Makevars.template -x COPYING -x INSTALL -x '*.orig' -x '*.rej' -x '*~' -x '*.temp' -x texinfo.tex -x ylwrap -x 'Makefile.in*' -x 'makefile.in*' -x configure -x gnome-doc-utils.make -x intltool.m4 -x omf.make -x xmldocs.make -x common -x m4 -x po -x autoregen.sh -x stamp-h.in origsrc/gst-editor-0.8.0/libs/gst/debug-ui/Makefile.am src/gst-editor-0.8.0/libs/gst/debug-ui/Makefile.am
--- origsrc/gst-editor-0.8.0/libs/gst/debug-ui/Makefile.am	2004-04-03 00:24:03.000000000 -0600
+++ src/gst-editor-0.8.0/libs/gst/debug-ui/Makefile.am	2006-08-15 16:25:23.940375000 -0500
@@ -1,10 +1,11 @@
-lib_LTLIBRARIES = libgstdebugui.la
+noinst_LTLIBRARIES = libgstdebugui.la
 
 libgstdebugui_la_SOURCES =	\
 	debug-ui.c
 
 libgstdebugui_la_CFLAGS = -DDATADIR="\"$(pkgdatadir)/\"" $(GST_EDITOR_CFLAGS)
-libgstdebugui_la_LDFLAGS = $(GST_EDITOR_LIBS)
+#libgstdebugui_la_LIBADD = $(top_builddir)/libs/gst/common/libgste-common.la $(GST_EDITOR_LIBS)
+libgstdebugui_la_LDFLAGS = -no-undefined
 
 libgstdebuguiincludedir = $(includedir)/@PACKAGE@-@VERSION@/gst/debug-ui
 libgstdebuguiinclude_HEADERS = debug-ui.h
diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x ltmain.sh -x 'config.*' -x depcomp -x install-sh -x missing -x mkinstalldirs -x autom4te.cache -x '*compile' -x 'intltool*.in' -x 'xml-i18n-*.in' -x '*.pyc' -x '*.mo' -x '*.gmo' -x ABOUT-NLS -x Makevars.template -x COPYING -x INSTALL -x '*.orig' -x '*.rej' -x '*~' -x '*.temp' -x texinfo.tex -x ylwrap -x 'Makefile.in*' -x 'makefile.in*' -x configure -x gnome-doc-utils.make -x intltool.m4 -x omf.make -x xmldocs.make -x common -x m4 -x po -x autoregen.sh -x stamp-h.in origsrc/gst-editor-0.8.0/libs/gst/editor/Makefile.am src/gst-editor-0.8.0/libs/gst/editor/Makefile.am
--- origsrc/gst-editor-0.8.0/libs/gst/editor/Makefile.am	2004-04-23 08:45:46.000000000 -0500
+++ src/gst-editor-0.8.0/libs/gst/editor/Makefile.am	2006-08-15 16:26:28.377875000 -0500
@@ -1,4 +1,4 @@
-lib_LTLIBRARIES = libgsteditor.la
+noinst_LTLIBRARIES = libgsteditor.la
 
 libgsteditor_la_SOURCES =	\
 	gsteditor.c		\
@@ -16,11 +16,13 @@
 	stockicons.c
 
 libgsteditor_la_CFLAGS = -DDATADIR="\"$(pkgdatadir)/\"" $(GST_EDITOR_CFLAGS)
-libgsteditor_la_LDFLAGS = $(GST_EDITOR_LIBS) \
+libgsteditor_la_LIBADD = \
 	$(top_builddir)/libs/gst/common/libgste-common.la \
 	$(top_builddir)/libs/gst/element-ui/libgstelementui.la \
 	$(top_builddir)/libs/gst/element-browser/libgstelementbrowser.la \
-	$(top_builddir)/libs/gst/debug-ui/libgstdebugui.la
+	$(top_builddir)/libs/gst/debug-ui/libgstdebugui.la \
+	$(GST_EDITOR_LIBS)
+libgsteditor_la_LDFLAGS = -no-undefined
 
 noinst_HEADERS =                \
         gsteditorproperty.h     \
diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x ltmain.sh -x 'config.*' -x depcomp -x install-sh -x missing -x mkinstalldirs -x autom4te.cache -x '*compile' -x 'intltool*.in' -x 'xml-i18n-*.in' -x '*.pyc' -x '*.mo' -x '*.gmo' -x ABOUT-NLS -x Makevars.template -x COPYING -x INSTALL -x '*.orig' -x '*.rej' -x '*~' -x '*.temp' -x texinfo.tex -x ylwrap -x 'Makefile.in*' -x 'makefile.in*' -x configure -x gnome-doc-utils.make -x intltool.m4 -x omf.make -x xmldocs.make -x common -x m4 -x po -x autoregen.sh -x stamp-h.in origsrc/gst-editor-0.8.0/libs/gst/element-browser/Makefile.am src/gst-editor-0.8.0/libs/gst/element-browser/Makefile.am
--- origsrc/gst-editor-0.8.0/libs/gst/element-browser/Makefile.am	2004-04-11 13:42:14.000000000 -0500
+++ src/gst-editor-0.8.0/libs/gst/element-browser/Makefile.am	2006-08-15 16:25:47.971625000 -0500
@@ -1,4 +1,4 @@
-lib_LTLIBRARIES = libgstelementbrowser.la
+noinst_LTLIBRARIES = libgstelementbrowser.la
 
 libgstelementbrowser_la_SOURCES =	\
 	browser.c			\
@@ -6,7 +6,8 @@
 	element-tree.c
 
 libgstelementbrowser_la_CFLAGS = -DDATADIR="\"$(pkgdatadir)/\"" $(GST_EDITOR_CFLAGS)
-libgstelementbrowser_la_LDFLAGS = $(GST_EDITOR_LIBS)
+#libgstelementbrowser_la_LIBADD = $(top_builddir)/libs/gst/common/libgste-common.la $(GST_EDITOR_LIBS)
+libgstelementbrowser_la_LDFLAGS = -no-undefined
 
 libgstelementbrowserincludedir = $(includedir)/@PACKAGE@-@VERSION@/gst/element-browser
 libgstelementbrowserinclude_HEADERS = browser.h caps-tree.h element-tree.h
diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x ltmain.sh -x 'config.*' -x depcomp -x install-sh -x missing -x mkinstalldirs -x autom4te.cache -x '*compile' -x 'intltool*.in' -x 'xml-i18n-*.in' -x '*.pyc' -x '*.mo' -x '*.gmo' -x ABOUT-NLS -x Makevars.template -x COPYING -x INSTALL -x '*.orig' -x '*.rej' -x '*~' -x '*.temp' -x texinfo.tex -x ylwrap -x 'Makefile.in*' -x 'makefile.in*' -x configure -x gnome-doc-utils.make -x intltool.m4 -x omf.make -x xmldocs.make -x common -x m4 -x po -x autoregen.sh -x stamp-h.in origsrc/gst-editor-0.8.0/libs/gst/element-ui/Makefile.am src/gst-editor-0.8.0/libs/gst/element-ui/Makefile.am
--- origsrc/gst-editor-0.8.0/libs/gst/element-ui/Makefile.am	2002-12-11 07:10:45.000000000 -0600
+++ src/gst-editor-0.8.0/libs/gst/element-ui/Makefile.am	2006-08-15 16:25:57.112250000 -0500
@@ -1,11 +1,12 @@
-lib_LTLIBRARIES = libgstelementui.la
+noinst_LTLIBRARIES = libgstelementui.la
 
 libgstelementui_la_SOURCES =		\
 	gst-element-ui.c		\
 	gst-element-ui-prop-view.c
 
 libgstelementui_la_CFLAGS = $(GST_EDITOR_CFLAGS)
-libgstelementui_la_LDFLAGS = $(GST_EDITOR_LIBS)
+#libgstelementui_la_LIBADD = $(GST_EDITOR_LIBS)
+libgstelementui_la_LDFLAGS = -no-undefined
 
 libgstelementuiincludedir = $(includedir)/@PACKAGE@-@VERSION@/gst/element-ui
 libgstelementuiinclude_HEADERS = gst-element-ui.h gst-element-ui-prop-view.h
diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x ltmain.sh -x 'config.*' -x depcomp -x install-sh -x missing -x mkinstalldirs -x autom4te.cache -x '*compile' -x 'intltool*.in' -x 'xml-i18n-*.in' -x '*.pyc' -x '*.mo' -x '*.gmo' -x ABOUT-NLS -x Makevars.template -x COPYING -x INSTALL -x '*.orig' -x '*.rej' -x '*~' -x '*.temp' -x texinfo.tex -x ylwrap -x 'Makefile.in*' -x 'makefile.in*' -x configure -x gnome-doc-utils.make -x intltool.m4 -x omf.make -x xmldocs.make -x common -x m4 -x po -x autoregen.sh -x stamp-h.in origsrc/gst-editor-0.8.0/pixmaps/Makefile.am src/gst-editor-0.8.0/pixmaps/Makefile.am
--- origsrc/gst-editor-0.8.0/pixmaps/Makefile.am	2002-10-05 13:18:33.000000000 -0500
+++ src/gst-editor-0.8.0/pixmaps/Makefile.am	2006-07-21 18:07:01.228500000 -0500
@@ -20,7 +20,7 @@
 CLEANFILES = $(BUILT_SOURCES)
 DISTCLEANFILES = $(BUILT_SOURCES)
 
-pixmaps.h: gdk-pixbuf-csource $(PIXMAPS)
+pixmaps.h: gdk-pixbuf-csource$(EXEEXT) $(PIXMAPS)
 	-rm -f pixmaps.h pixmaps-tmp
 	for i in $(PIXMAPS); do \
 	   ./gdk-pixbuf-csource --raw --build-list \
diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x ltmain.sh -x 'config.*' -x depcomp -x install-sh -x missing -x mkinstalldirs -x autom4te.cache -x '*compile' -x 'intltool*.in' -x 'xml-i18n-*.in' -x '*.pyc' -x '*.mo' -x '*.gmo' -x ABOUT-NLS -x Makevars.template -x COPYING -x INSTALL -x '*.orig' -x '*.rej' -x '*~' -x '*.temp' -x texinfo.tex -x ylwrap -x 'Makefile.in*' -x 'makefile.in*' -x configure -x gnome-doc-utils.make -x intltool.m4 -x omf.make -x xmldocs.make -x common -x m4 -x po -x autoregen.sh -x stamp-h.in origsrc/gst-editor-0.8.0/src/Makefile.am src/gst-editor-0.8.0/src/Makefile.am
--- origsrc/gst-editor-0.8.0/src/Makefile.am	2004-09-26 04:54:55.000000000 -0500
+++ src/gst-editor-0.8.0/src/Makefile.am	2006-08-15 16:27:05.049750000 -0500
@@ -1,17 +1,18 @@
 bin_PROGRAMS = gst-editor gst-launch-gui gst-inspect-gui
 
 gst_editor_SOURCES = editor.c
-gst_editor_LDFLAGS = $(top_builddir)/libs/gst/editor/libgsteditor.la $(GST_EDITOR_LIBS)
+gst_editor_LDADD = $(top_builddir)/libs/gst/editor/libgsteditor.la $(GST_EDITOR_LIBS)
 gst_editor_CFLAGS = $(GST_EDITOR_CFLAGS) -DDATADIR=\"$(gladedir)/\" -I$(top_srcdir)/libs
 
-gst_launch_gui_LDADD = $(GST_LIBS) $(GNOME_LIBS) \
+gst_launch_gui_LDADD = \
 	 $(top_builddir)/libs/gst/common/libgste-common.la \
 	 $(top_builddir)/libs/gst/element-ui/libgstelementui.la \
-	 $(top_builddir)/libs/gst/debug-ui/libgstdebugui.la
+	 $(top_builddir)/libs/gst/debug-ui/libgstdebugui.la \
+	 $(GST_LIBS) $(GNOME_LIBS)
 gst_launch_gui_CFLAGS = $(GST_CFLAGS) $(GNOME_CFLAGS) -I$(top_srcdir)/libs \
 	-DPIXMAP_DIR=\"$(datadir)/pixmaps/\"
 
-gst_inspect_gui_LDADD = $(GST_EDITOR_LIBS) $(top_builddir)/libs/gst/element-browser/libgstelementbrowser.la $(top_builddir)/libs/gst/debug-ui/libgstdebugui.la $(top_builddir)/libs/gst/common/libgste-common.la 
+gst_inspect_gui_LDADD = $(top_builddir)/libs/gst/element-browser/libgstelementbrowser.la $(top_builddir)/libs/gst/debug-ui/libgstdebugui.la $(top_builddir)/libs/gst/common/libgste-common.la  $(GST_EDITOR_LIBS)
 gst_inspect_gui_CFLAGS = $(GST_EDITOR_CFLAGS) -I$(top_srcdir)/libs
 
 gladedir = $(datadir)/gst-editor
